packages/server/safetyPatchRes.ts

```diff
@@ -37,6 +37,15 @@ const safetyPatchRes = (res: HttpResponse) => {
     return res._close()
   }
 
+  res._cork = res.cork
+  res.cork = (cb: () => void) => {
+    if (res.done) {
+      console.log(`uWS DEBUG: Called cork after done`)
+      return
+    }
+    res._cork(cb)
+  }
+
```

Comment:
-1 all methods from `HttpResponse` interface (including getters) potentially throw this error
https://github.com/uNetworking/uWebSockets.js/blob/master/src/HttpResponseWrapper.h#L26-L34
Of the missing ones we at lease use `upgrade` and `getRemoteAddressAsText`.

Reply:
great catch! i forgot we call those after an `await` will add.
while i'm in here, i'm gonna break done between aborted & done so we get more meaningful errors